aea/test_tools/generic.py

```diff
@@ -55,8 +55,8 @@ def read_envelope_from_file(file_path: str):
     with open(Path(file_path), "rb+") as f:
         lines.extend(f.readlines())
 
-    enforce(len(lines) == 2, "Did not find two lines.")
-    line = lines[0] + lines[1]
+    enforce(len(lines) == 3, "Did not find three lines.")
+    line = lines[0] + lines[1] + lines[2]
```

Comment:
See above; although we could leave this if we are sure it is always 3 lines for default protocol..hacky I know

Reply:
with the new Proto definition it seems 2 are enough, see changes